File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/transforms/go/fixture_real.go
package main

import "fmt"
import "strconv"

func main() {
	var state int64 = 7
	var sum int64 = 0
	var largest int64 = 0
	var evens int64 = 0
	var joined string = ""
	var i int64 = 0
	for i = 0; i < 50; i = i + 1 {
		state = (state * 48271) % 2147483647
		var term int64 = state % 1000
		sum = sum + term
		if term > largest {
			largest = term
		}
		if term % 2 == 0 {
			evens = evens + 1
		}
		if i < 5 {
			if i > 0 {
				joined = joined + "-"
			}
			joined = joined + strconv.FormatInt(term, 10)
		}
	}
	fmt.Println("SUM " + strconv.FormatInt(sum, 10))
	fmt.Println("MAX " + strconv.FormatInt(largest, 10))
	fmt.Println("EVENS " + strconv.FormatInt(evens, 10))
	fmt.Println("JOINED " + joined)
}

File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/transforms/java/fixture_real.java
public class Main {
    public static void main(String[] args) {
        long state = 7;
        long sum = 0;
        long max = 0;
        int evens = 0;
        String joined = "";
        for (int i = 0; i < 50; i++) {
            state = (state * 48271) % 2147483647;
            long term = state % 1000;
            sum = sum + term;
            if (term > max) {
                max = term;
            }
            if (term % 2 == 0) {
                evens = evens + 1;
            }
            if (i < 5) {
                if (i == 0) {
                    joined = joined + term;
                } else {
                    joined = joined + "-" + term;
                }
            }
        }
        System.out.println("SUM " + sum);
        System.out.println("MAX " + max);
        System.out.println("EVENS " + evens);
        System.out.println("JOINED " + joined);
    }
}
